vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/SetVariableSteps.java

```diff
@@ -82,7 +82,7 @@ public SetVariableSteps(IWebDriverProvider webDriverProvider, ISoftAssert softAs
      * </ul>
      * @param variable A name under which the value should be saved
      */
-    @When("I get the value from the URL and set it to the '$scopes' variable '$variable'")
+    @When("I get value from URL and set it to $scopes variable `$variable`")
```

Comment:
I would recommend
- to deprecate this step (to be removed in 0.7.0)
- to use a combination of expressions and another step initializing variable
- to add a test for old deprecated step (if it's missing) and for the new approach